bin/tuning.py

```diff
@@ -47,37 +39,54 @@ def SleepRayTune(config, cfg: DictConfig, num_epochs, num_gpu):
         progress_bar_refresh_rate=0,
         callbacks=[
             TuneReportCallback(
-                {"loss": "val_loss"},
-                on="validation_end",
+                {
+                    "loss": "val_loss",
+                    "star_count_acc": "val_acc_counts",
+                    "galaxy_counts_acc": "val_gal_counts",
+                    "locs_median_mse": "val_locs_median_mse",
+                    "fluxes_avg_err": "val_fluxes_avg_err",
+                },
+                on="validation_step",
             )
         ],
     )
     trainer.fit(model, datamodule=dataset)
 
 
 @hydra.main(config_path="../config", config_name="config")
-# model=m2, dataset=m2, training=m2 in terminal
-def main(cfg: DictConfig, num_epochs=100, gpus_per_trial=1):
+# model=tuning, dataset=m2, training=tuning optimizer=m2 in terminal
+# TODO: Maybe expose the config for search space outside of main either as argument or with Hydra 
+def main(cfg: DictConfig, num_epochs=200, gpus_per_trial=1):
```

Comment:
it would be nice if these two parameters were part of the config as well. I think you are discussing this already in the comments above.

Reply:
We are working on collating the above two parameters, the total number of GPU for tuning, and the search space in a single config file.

Reply:
excellent, thanks!